gcp_variant_transforms/libs/preprocess_reporter.py

```diff
@@ -204,24 +205,27 @@ def _generate_conflicting_headers_lines(
                     num=1 type=Integer   file2
   """
   content_lines = []
-  for field_id, definitions_to_files_map in conflicts.iteritems():
+  sorted_field_ids = sorted(conflicts.keys())
+  for field_id in sorted_field_ids:
```

Comment:
in here you can also do: `for field_id in sorted(conflicts.keys()):` and the same thing below for definitions.